src/scripts/main.js

```diff
@@ -1,3 +1,188 @@
+  createInput('number', 'Age'),
+  createInput('number', 'Salary'),
+  submitButton,
+];
+
+newForm.append(...allInputsForForm);
+mainTable.after(newForm);
+
+function createInput(typeInput, nameInput) {
+  const newLabel = document.createElement('label');
+  const newInput = document.createElement('input');
+
+  newLabel.append(`${nameInput}:`);
+  newInput.name = nameInput.toLowerCase();
+  newInput.type = typeInput.toLowerCase();
+  newInput.dataset.qa = nameInput.toLowerCase();
+  newInput.required = true;
+  newLabel.append(newInput);
+
+  return newLabel;
+}
+
+function createSelect(array, nameForm) {
+  const newSelect = document.createElement('select');
+  const newLabel = document.createElement('label');
+
+  newLabel.append(`${nameForm}:`);
+
+  array.forEach((value) => {
+    const newOption = document.createElement('option');
+
+    newOption.value = value;
+    newOption.innerText = value;
+    newSelect.append(newOption);
+  });
+
+  newSelect.name = nameForm.toLowerCase();
+  newSelect.dataset.qa = nameForm.toLowerCase();
+  newLabel.append(newSelect);
+
+  return newLabel;
+}
+
+function sortForTable(sortTable, column) {
+  const tbody = sortTable.querySelector('tbody');
+
+  return Array.from(tbody.children).sort((row1, row2) => {
+    const value1 = row1.children[column].textContent;
+    const value2 = row2.children[column].textContent;
+
+    if (!/[0-9]/.test(value1)) {
+      return value1.localeCompare(value2);
+    } else if (/[0-9]/.test(value1)) {
+      return value1.replace(/\D/g, '') - value2.replace(/\D/g, '');
+    }
+  });
+}
+
+function createMessage(typeNotification, info) {
+  const notification = document.createElement('div');
+  const titleNotification = document.createElement('h2');
+  const textNotification = document.createElement('p');
+  const oldMessage = document.querySelector('.notification');
+
+  if (oldMessage) {
+    oldMessage.remove();
+  }
+
+  notification.dataset.qa = 'notification';
+  titleNotification.classList.add('title');
+
+  if (typeNotification === 'error') {
+    notification.classList.add('notification', 'error');
+    titleNotification.innerText = 'Incorrect data entered.';
+
+    if (info === 'age') {
+      textNotification.innerText = `Please check the ${info}. Minimum age 18, maximum 90.`;
+    } else {
+      textNotification.innerText = `Please check the ${info} and make corrections.`;
+    }
+  } else if (typeNotification === 'success') {
+    notification.classList.add('notification', 'success');
+    titleNotification.innerText = 'Success!';
+    textNotification.innerText = 'Data added successfully';
+  }
+
+  notification.append(titleNotification, textNotification);
+  document.body.append(notification);
+
+  window.setTimeout(() => notification.remove(), 3000);
+}
+
+function validationSalary(value) {
+  const clearNum = value.replace(/\D/g, '');
+  const formatedValue = parseFloat(clearNum).toLocaleString('en-US');
+
+  return '$' + formatedValue;
```

Comment:
The `validationSalary` function may not format the salary correctly if the input is not a valid number. Ensure that the input is validated and formatted correctly to avoid potential issues.